features/scaffold-block.feature

```diff
@@ -5,8 +5,8 @@ Feature: WordPress block code scaffolding
     Given I run `wp scaffold plugin movies`
     And I run `wp plugin path movies --dir`
     And save STDOUT as {PLUGIN_DIR}
-    Given I run `wp theme install p2 --activate`
-    And I run `wp theme path p2 --dir`
+    Given I try `wp theme install twentytwelve --activate --force`
```

Comment:
I'd expect this to remain as `run`. What errors are thrown?

Reply:
Changed `try` to `run`.